.fixtures.yml

```diff
@@ -29,27 +12,15 @@ fixtures:
     elasticsearch:
       repo: https://github.com/simp/puppet-elasticsearch
       ref: 5.5.0
-    haveged: https://github.com/simp/pupmod-simp-haveged
     iptables: https://github.com/simp/pupmod-simp-iptables
     java:
       repo: https://github.com/simp/puppetlabs-java
       ref: 2.4.0
-    logrotate: https://github.com/simp/pupmod-simp-logrotate
-    oddjob: https://github.com/simp/pupmod-simp-oddjob
     pam: https://github.com/simp/pupmod-simp-pam
     pki: https://github.com/simp/pupmod-simp-pki
     rsync: https://github.com/simp/pupmod-simp-rsync
-    rsyslog: https://github.com/simp/pupmod-simp-rsyslog
     simp_apache: https://github.com/simp/pupmod-simp-apache
-    simp_options: https://github.com/simp/pupmod-simp-simp_options
-    simpcat: https://github.com/simp/pupmod-simp-simpcat
     simplib: https://github.com/simp/pupmod-simp-simplib
-    stdlib:
-      repo: https://github.com/simp/puppetlabs-stdlib
-      ref: 4.24.0
-    tcpwrappers: https://github.com/simp/pupmod-simp-tcpwrappers
-    yum:
```

Comment:
yum is here because at one point, we needed to pin the version of elasticsearch via the elasticsearch module.  That module uses the yum module.  That said, we can easily add it back in when we need it.